packages/web/src/pages/sign-up-page/components/layout.tsx

```diff
@@ -11,73 +13,109 @@ import {
 
 import { useMedia } from 'hooks/useMedia'
 
-import { ContinueFooter } from './ContinueFooter'
+import { ContinueFooter, ContinueFooterProps } from './ContinueFooter'
 
 const messages = {
   continue: 'Continue'
 }
 
 type PageProps = FlexProps & {
   as?: ComponentType<any>
+  centered?: boolean
 }
 
 export const Page = (props: PageProps) => {
+  const { centered, children, as, ...other } = props
   const { isMobile } = useMedia()
+
+  const childrenArray = Children.toArray(children)
+  const footer = childrenArray.pop()
```

Comment:
Hmm. This is very clever, but feels a bit hacky to make assumptions about children. What about `props.footer` or

```
<Page>
  <Page.Content></...>
  <Page.Footer></...>
</Page>
```

Reply:
yeah it is pretty hacky, agreed... i didnt want it as props.footer for the preception that it needs to be in formik context, but we can ensure that pretty easily